README.md

```diff
@@ -92,15 +99,18 @@ necessary to do the OAuth Flow again. You can just use a `TokenConfiguration`.
 
 ```swift
 let token = // get your token from your keychain, user defaults (not recommended) etc.
-let config = TokenConfiguration(token)
-Octokit(config).user("octocat") { response in
+let config = TokenConfiguration()
+config.accessToken = token
+
+let _ = Octokit(config).user("octocat") { response in
   switch response {
-  case .Success(let user):
-    println(user.login)
-  case .Failure(let error):
-    println(error)
+  	case .success(let user):
```

Comment:
Can you check the indentation here? Seems to me that these are indented too much